charts/incubator/privatebin/questions.yaml

```diff
@@ -0,0 +1,544 @@
+          schema:
+            type: boolean
+            default: false
+            show_subquestions_if: true
+            subquestions:
+              - variable: type
+                description: "Please specify type of workload to deploy"
+                label: "(Advanced) Controller Type"
+                schema:
+                  type: string
+                  default: "deployment"
+                  required: true
+                  enum:
+                    - value: "deployment"
+                      description: "Deployment"
+                    - value: "statefulset"
+                      description: "Statefulset"
+                    - value: "daemonset"
+                      description: "Daemonset"
+              - variable: replicas
+                description: "Number of desired pod replicas"
+                label: "Desired Replicas"
+                schema:
+                  type: int
+                  default: 1
+                  required: true
+              - variable: strategy
+                description: "Please specify type of workload to deploy"
+                label: "(Advanced) Update Strategy"
+                schema:
+                  type: string
+                  default: "Recreate"
+                  required: true
+                  enum:
+                    - value: "Recreate"
+                      description: "Recreate: Kill existing pods before creating new ones"
+                    - value: "RollingUpdate"
+                      description: "RollingUpdate: Create new pods and then kill old ones"
+                    - value: "OnDelete"
+                      description: "(Legacy) OnDelete: ignore .spec.template changes"
+# Include{controllerExpert}
+
+  - variable: privatebin
+    group: "Container Configuration"
+    label: "PrivateBin Custom Configuration"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: main
+          label: "Main Configuration"
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: name
+                label: "name"
+                description: "Set a project name to be displayed on the website"
+                schema:
+                  type: string
+                  default: ""
+              - variable: basepath
+                label: "basepath"
+                description: "The full URL, with the domain name and directories that point to the PrivateBin files. This URL is essential to allow Opengraph images to be displayed on social networks."
+                schema:
+                  type: string
+                  default: ""
+              - variable: discussion
+                label: "discussion"
+                description: "Enable or disable the discussion feature"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
```

Comment:
required: true on booleans will allow only value of `true`.
eg. You can't set this to false.

Reply:
Do I use string field + enum instead?

Reply:
If you want the user to select either false or true.
Just remove the `required` otherwise you force him to `true`